point.py

```diff
@@ -0,0 +1,108 @@
+import random
+import math
+
+
+class Point:
+    """
+    For whatever reason, cannot get import to work, just moved functionality
+    into this file instead.
+    """
```

Comment:
Check out the general discussion forum post about imports and let me know if that helps.  If not, respond to that post and we can debug.